### PR TITLE
fix dupes in airswap dex trades

### PR DESCRIPTION
**Pull request by jeff-dude**
fyi @Hosuke, i simplified the query & fixed the approach for unique key. comment below where the dupes were introduced.

### CODE REVIEW

File: models/airswap/ethereum/airswap_ethereum_trades.sql [hunk 1]
```diff
@@ -1,186 +1,146 @@
...
+    ON dexs.tx_hash = tx.hash
+    {% if not is_incremental() %}
+    AND tx.block_time >= '{{project_start_date}}'
+    {% endif %}
+    {% if is_incremental() %}
+    AND tx.block_time = date_trunc("day", now() - interval '1 week')
+    {% endif %}
+LEFT JOIN {{ ref('tokens_erc20') }} erc20a
+    ON erc20a.contract_address = dexs.token_bought_address
+    AND erc20a.blockchain = 'ethereum'
```
Comment (jeff-dude) [inline models/airswap/ethereum/airswap_ethereum_trades.sql:122]: missing the blockchain filter on the two joins to erc20 tokens was causing the dupes

**Review by Hosuke — APPROVED** [review]
It's a huge improvement, and thanks for fixing the dupes I produced from the last version.
Sorry I just knew that there are [records with the same `contract_address` but from different `blockchain` in tokens.erc20](https://dune.com/queries/1385967).

File: models/airswap/ethereum/airswap_ethereum_schema.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ models:
             - version
             - tx_hash
             - evt_index
-            - trade_id
```
Comment (Hosuke) [inline models/airswap/ethereum/airswap_ethereum_schema.yml:23]: line 94-96
`- &trade_id
        name: trade_id
        description: "Ranked record id for the same trade event"
`
also can be removed.

**Review by soispoke — APPROVED** [review]
lgtm

**Comment by jeff-dude** [comment]
> It's a huge improvement, and thanks for fixing the dupes I produced from the last version. Sorry I just knew that there are [records with the same `contract_address` but from different `blockchain` in tokens.erc20](https://dune.com/queries/1385967).
> 
> The dupes I produced actually majorly comes from trades with 1INCH and CHI, they are tokens having the same contract address on multiple chains. You are truly insightful.

glad this helps!

File: models/airswap/ethereum/airswap_ethereum_schema.yml (hunk 2 above)
Comment (jeff-dude) [reply models/airswap/ethereum/airswap_ethereum_schema.yml:23]: thank you!

**Pull request merged**